lib/rx/configuration.rb

```diff
@@ -20,7 +20,7 @@ def app_token
     end
 
     def base_path
-      "#{Settings.mhv.rx.host}/mhv-api/patient/v1/"
+      "#{Settings.mhv.rx.host}/patient/v1/"
```

Comment:
I believe there would be a negative impact to changing this base path that connects us with the established external service.  For example, not being able to connect with their API.

Based [on the cassettes](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr_cassettes/rx_client/prescriptions/gets_a_list_of_all_prescriptions.yml#L5), I think we'll want to keep this base path as is.

Reply:
@hpjaj I updated the cassettes to match, but I think I'm following what you're saying that it should match the live endpoint's URL that includes that path. I removed it because I thought I saw it absent in other places in the code, but I was probably mistaken.